src/utils/level.ml

```diff
@@ -0,0 +1,48 @@
+type t = int
```

Comment:
It may be a good idea to expose the comparison instead of using `(<) : 'a -> 'a -> bool` though.